packages/chart/src/CdcChart.tsx

```diff
@@ -1101,6 +1101,11 @@ export default function CdcChart({ configUrl, config: configObj, isEditor = fals
   // Prevent render if loading
   let body = <Loading />
 
```

Comment:
nit: classes in this area could follow the ['className', 'className-2'].join(' ') pattern. There might also be a new layout container to use here, but that could be out of scope for this ticket.